addons/skin.estuary/xml/GameOSD.xml

```diff
@@ -206,6 +206,12 @@
 								<icon>osd/fullscreen/buttons/play.png</icon>
 								<onclick>Play</onclick>
 							</item>
+							<item id="2107">
```

Comment:
Any reason you moved this block up?

Reply:
I believe the "Save / Load" option is going to be used more often then the rest of the menu so moving it just below "Pause / Resume" makes sense to me.

Reply:
I like how Reset is now next to Stop, so I think it's a good change.

Reply:
We can re-order the IDs if they're not used anywhere, and I'm pretty sure that's the case because they all take effect via the `<onclick>` property. I remember that being one of my goals when I made this.

Reply:
So do you want me to re-order the IDs? There were missing IDs already before this patch (2101, 2102, 2103, 2107, 2108), so should I keep it or renumber it from 2101 to 2105?

Btw. What about the Estouchy skin? I think there were no updates regarding the Savestate manager at all.